packages/person/src/person-avatar/element.ts

```diff
@@ -55,33 +55,33 @@ export class PersonAvatarElement extends PersonElement implements PersonAvatarEl
   /**
    * Returns the badge color for the current presence
    */
-  protected getRenderClasses(accountType?: string): ClassInfo {
+  protected getRenderClasses(accountType?: PersonAccountTypes): ClassInfo {
     return {
-      'fwc-person-avatar__employee': accountType === PersonAccountType.Employee,
-      'fwc-person-avatar__consultant': accountType === PersonAccountType.Consultant || PersonAccountType.Enterprise,
-      'fwc-person-avatar__external': accountType === PersonAccountType.External,
-      'fwc-person-avatar__external-hire': accountType === PersonAccountType.ExternalHire,
```

Comment:
If you want to remove enum, change only that, don't change class names

Reply:
Thanks for the save, did'nt see you had changes to the enum type